src/ragger/conftest/configuration.py

```diff
@@ -0,0 +1,30 @@
+from typing import Dict, Any
+
+# The following parameters are required.
+# Please set their values in your 'conftest.py' file
+REQUIRED_CONFIGURATION: Dict[str, Any] = {
+
+    # Use the real name of your application
+    # example: "boilerplate"
```

Comment:
Maybe something like: It is used to compute the elf name with template `<APP_NAME>_<DEVICE_NAME>.elf`

Reply:
Done